cartofacade/capi.go

```diff
@@ -54,13 +54,13 @@ type CartoInterface interface {
 	stop() error
 	terminate() error
 	addSensorReading(string, []byte, time.Time) error
-	getPosition() (GetPosition, error)
+	getPosition() (PositionInfo, error)
 	getPointCloudMap() ([]byte, error)
 	getInternalState() ([]byte, error)
 }
 
-// GetPosition holds values returned from c to be processed later
-type GetPosition struct {
+// PositionInfo holds values returned from c to be processed later
+type PositionInfo struct {
```

Comment:
Need to change this so that I can call the function in the API `GetPosition`, open to additional name suggestions

Reply:
I don't understand that. You should be able to have a function called GetPosition & a struct called GetPosition. If not, lets call it GetPositionResponse.

Reply:
I mustve misunderstood the error I was getting earlier, it is working now reverted